infomaniak-build-tools/linux/build-ci.sh

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash
+
+ulimit -n 4000000
+
+set -xe
+
+export QTDIR="~/Qt/6.2.3/gcc_64"
```

Comment:
We may define `QTDIR` based on `QT_BASE_DIR`.

Reply:
Done in latest commit